src/drivers/intel/cavs/ipc.c

```diff
@@ -56,6 +56,24 @@
 
 extern struct ipc *_ipc;
 
+#if defined(CONFIG_DEBUG_IPC_COUNTERS)
```

Comment:
`#if CONFIG_DEBUG_IPC_COUNTERS` is better.

Reply:
@jajanusz 
Updated